setup.py

```diff
@@ -45,17 +45,30 @@ def run(self):
         self.execute(self.updatePOT, ())
         self.execute(self.generate_mo_files, ())
         self.execute(self.buildInkscapeExt, ())
-        try:
-            path = check_output(["inkscape", "-x"]).decode().strip()
-            if not os.access(path, os.W_OK): # Can we install globaly
-                # Not tested on Windows and Mac
-                path = os.path.expanduser("~/.config/inkscape/extensions")
+
+        if 'FAKEROOTKEY' in os.environ:
+            # we are probably running under fakeroot
+            # so we are probably building a Debian package
+            # let us define a simple path!
+            path="/usr/share/inkscape/extensions"
             self.distribution.data_files.append(
                 (path,
                  [i for i in glob.glob(os.path.join("inkex", "*.inx"))]))
             self.distribution.data_files.append((path, ['scripts/boxes']))
-        except CalledProcessError:
-            pass # Inkscape is not installed
+        else:
+            # we are surely not building a Debian package
+            # then here is the default behavior:
+            try:
+                path = check_output(["inkscape", "-x"]).decode().strip()
+                if not os.access(path, os.W_OK): # Can we install globaly
+                    # Not tested on Windows and Mac
+                    path = os.path.expanduser("~/.config/inkscape/extensions")
+                self.distribution.data_files.append(
```

Comment:
Guess adding the files to the self.distribution does not really need to be in the try except block. So we could have only one place in the code that does that. Probably by setting path to None and using another if block.

Reply:
Never mind I'll just merge as is and fix that later.